docs/source/examples/candu.ipynb

```diff
@@ -0,0 +1,1113 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f45240e7550>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_args = {\n",
+    "    'width': (2*calendria_or, 2*calendria_or),\n",
+    "    'colors': {\n",
+    "        fuel: 'black',\n",
```

Comment:
They are actually only used for the last plot -- I'll just move the color specification there.